pkg/app/piped/executor/cloudrun/cloudrun.go

```diff
@@ -153,6 +153,10 @@ func waitRevisionReady(ctx context.Context, client provider.Client, revisionName
 
 	doCheck := func() (bool, error) {
 		rvs, err := client.GetRevision(ctx, revisionName)
+		// NotFound should be a retriable error.
+		if err == provider.ErrRevisionNotFound {
+			return true, err
```

Comment:
```suggestion
			return true, nil
```
Should it be like this? or stage is treated as failed, isn't it?

Reply:
In fact, nope, but I feel it's better.
https://github.com/pipe-cd/pipecd/pull/4092/files#diff-919ebf51d290a0c8fedd3995c66ccf085a3348f4e6bfde75ea102e1c4d188a19L198

Reply:
No, that should be returned to be included in the log message.

Reply:
Because that is a retriable error.

Reply:
Ah okay, at L212 👍